mailer.js

```diff
@@ -226,10 +236,7 @@ module.exports = function (log) {
 
   Mailer.prototype.newSyncDeviceEmail = function (message) {
     log.trace({ op: 'mailer.newSyncDeviceEmail', email: message.email, uid: message.uid })
-    var query = {
-      email: message.email
-    }
-    var link = this.initiatePasswordResetUrl + '?' + qs.stringify(query)
+    var link = this.initiatePasswordChangeUrl
 
```

Comment:
Good idea. I don't think content-server does anything with it on this page yet, but there's no harm in adding it here so we could autofill the login form if they aren't already logged in.